src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Color;
+  color : Shape;
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color : Color,
+    public a:number,
+    public b:number,
+    public c:number,
+  ) {
+    const max = Math.max(this.a, this.b, this.c);
 
-export class Circle {
+    if ((max >= (a + b + c - max)) || (a <= 0) || (b <= 0) || (c <= 0)) {
+      throw new Error(`sides ${this.a},
+        ${this.b} and ${this.c} can't form a triangle`);
+    }
+  }
 
+  getArea(): number {
+    const semiperimeter = (this.a + this.b + this.c) / 2;
+    const underSqrt = semiperimeter
+      * (semiperimeter - this.a)
+      * (semiperimeter - this.b) * (semiperimeter - this.c);
+    const AreaLong = Math.sqrt(underSqrt);
+    const Area = +((AreaLong).toFixed(2));
```

Comment:
```suggestion
    const areaLong = Math.sqrt(underSqrt);
    const area = +((AreaLong).toFixed(2));
```
There is no need to capitalize the first letter